math/gnudatalanguage/Portfile

```diff
@@ -18,98 +17,59 @@ mpi.setup
 
 categories                  math science
 maintainers                 {takeshi @tenomoto}
-license                     GPL-2
-platforms                   darwin
+license                     GPL-2+
 description                 a free IDL combatible incremental compiler
 long_description            A free IDL (Interactive Data Language) compatible \
                             incremental compiler (i.e. runs IDL programs).
-master_sites                ${github.homepage}/archive:gdl \
-                            http://www.soest.hawaii.edu/pwessel/gshhg:gshhs
-set gdlsrc          v${version}${extract.suffix}
-set dataversion     2.3.7
-set gshhsdata       gshhg-bin-${dataversion}.zip
-distfiles           ${gdlsrc}:gdl
-worksrcdir          gdl-${version}
-
-checksums           ${gdlsrc} \
-                    rmd160  e83bbdb6140211d4b4010208e836608ef6653e66 \
-                    sha256  0dc5c5a397d46002e763d24dba707f0aa55ba404062b4ba5d4c217af8a7da024 \
-                    size    32650313
+
+distname                    ${github.project}-${github.tag_prefix}${version}
+
+checksums           rmd160  6175c3d407a8c5aa4098bf9283025b1b3cad83e8 \
+                    sha256  744ed3abcdc5e1bbf31147a8a0c21c33662f200b6096ee3d3adedd160a3a9662 \
+                    size    33813887
 
 compiler.cxx_standard       2011
 
-depends_build-append        port:pkgconfig
+set proj_ver                9
+
+depends_build-append        path:bin/pkg-config:pkgconfig \
+                            path:share/pkgconfig/eigen3.pc:eigen3 \
+                            port:qhull
```

Comment:
no linkage to the "qhull" library?

Reply:
Nope. The C++ version of it uses a [static version](https://github.com/qhull/qhull/blob/d1c2fc0caa5f644f3a0f220290d4a868c68ed4f6/CMakeLists.txt#L451).